app/src/main/res/values/strings.xml

```diff
@@ -92,4 +92,9 @@
     <string name="invalid_email_format">Invalid email format!</string>
     <string name="least_6_characters">Please enter at least 6 characters!</string>
     <string name="confirm_password_does_not_match">Confirm password does not match!</string>
+    <string name="msg_error_init">Sorry, you can not init this audio. Please try again later!</string>
+    <string name="msg_error_start">Sorry, an error has occurred. Please try again later!</string>
+    <string name="description_play_audio">Play the detail audio</string>
+    <string name="description_avatar">Image post user</string>
```

Comment:
FATAL: [MissingTranslation] "`description_avatar`" is not translated in "de" (German), "fr" (French), "ru" (Russian)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)